app/assets/stylesheets/taxon_ranges/taxon.css

```diff
@@ -0,0 +1 @@
+input.ui-chooser-input { max-width:400px; }
```

Comment:
Since this has nothing to do with the taxon the range is associated with, how about renaming this file to `form.css`. Also needs a newline at the end.